interfaces/builtin/adb_support.go

```diff
@@ -119,6 +119,7 @@ var adbSupportVendors = map[int]string{
 	0x2a45: "MEIZU",
 	0x2a47: "BQ",
 	0x2a49: "UNOWHY",
+	0x2a70: "OnePlus",
```

Comment:
Random pages I looked at seem to agree about the Vendor ID. Some findings:
https://review.lineageos.org/c/LineageOS/android_device_oneplus_oneplus3/+/30185/1/rootdir/etc/init.qcom.usb.rc
http://turlucode.com/linux-android-adb-fix-permissions/
https://usb-ids.gowdy.us/read/UD/2a70